docs/decisions/decision_power_magg-e_v1.md

```
@@ -0,0 +1,83 @@
+
+---
+
+## 4. Chosen Configuration & Power Delivery Mapping
+
+| Power Bank       | Output Port      | Output Voltage / Max | Avg Output Power | Peak Output Power | Target Device              | Input Voltage / Req. | Avg Input Power | Peak Input Power | Headroom (Avg) | Headroom (Peak) |
+|------------------|------------------|-----------------------|------------------|-------------------|----------------------------|-----------------------|------------------|------------------|----------------|-----------------|
+| NB7102 (~440g)   | DC2              | 19.5V @ 3A (58.5W max)    | 17.5 W           | 23.5 W            | Pi5 + Camera + WiFi (via converter) | 5V (via buck)         | 17.5 W           | 23.5 W           | 54.5 W (311%)   | 48.5 W (206%)    |
+| NB7102 (~440g)   | USB-A            | 5V @ 2.4A (12W max)   | 1.5 W            | 5.0 W             | 2× Servo Gimbal            | 5V                    | 1.5 W            | 5.0 W            | 10.5 W (700%)  | 7.0 W (140%)    |
+| YB120300 (~200g) | DC barrel        | 12V @ 3A (36W max)    | 5.5 W            | 18.0 W            | Pi2 + BrickPi              | 12V                   | 5.5 W            | 18.0 W           | 30.5 W (554%)  | 18.0 W (100%)   |
+
+*Headroom = Output capacity minus Input requirement*
```

markdownlint-cli2 (0.17.2)

Emphasis used instead of a heading
null

(MD036, no-emphasis-as-heading)